leaf-ui/js/onFunctions.js

```diff
@@ -472,37 +456,37 @@ $('#model-cur-btn').on('click', function() {
  */
 function revertNodeValuesToInitial() {
     // reset values
-    for (var i = 0; i < graph.elementsBeforeAnalysis.length; i++) {
-		var value = graph.elementsBeforeAnalysis[i]
-		updateNodeValues(i, value, "toInitModel");
-	}
-
-	var elements = graph.getElements();
-	var curr;
-	for (var i = 0; i < elements.length; i++) {
-		curr = elements[i].findView(paper).model;
-
-		if (curr.attributes.type !== 'basic.Goal' &&
-			curr.attributes.type !== 'basic.Task' &&
-			curr.attributes.type !== 'basic.Softgoal' &&
-			curr.attributes.type !== 'basic.Resource') {
-			continue;
-		}
-
-		var intention = model.getIntentionByID(curr.attributes.nodeID);
-
-		var initSatVal = intention.getInitialSatValue();
-		if (initSatVal === '(no value)') {
-            curr.attr('.satvalue/text', '');
-
-		} else {
-            curr.attr('.satvalue/text', satisfactionValuesDict[initSatVal].satValue);
-		}
-        //curr.attr({text: {fill: 'black'}});
-        curr.attr({text: {fill: 'black',stroke:'none','font-weight' : 'normal','font-size': 10}});
-	}
-    // Remove slider
-    removeSlider();
+    // for (var i = 0; i < graph.elementsBeforeAnalysis.length; i++) {
```

Comment:
I can uncomment it - It is no longer relevant to my branch. I think Manar and Angela are working on a fix for the error this function throws currently.

Reply:
I was wondering if it is dead code that should be removed?

Reply:
I'm honestly not sure - I think the goal of the function remains necessary, the implementation is now outdated though. I'd want to replace this function rather than delete it and potentially lose it but up to you